helpers/requests/requests.go

```diff
@@ -0,0 +1,82 @@
+package requests
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+	"time"
+)
+
+// constant request methods
+const (
+	DELETE = "DELETE"
+	GET    = "GET"
+	POST   = "POST"
+)
+
+// Timeout request set to 5 minutes
+const (
+	RequestTimeout = time.Duration(5) * time.Minute
+)
+
+// Request builds a request and calls execute to send request
+func Request(method, url, authorization, unitSystem string, body io.Reader) (http.Response, error) {
+
+	// Build Request
+	req, reqErr := http.NewRequest(method, url, body)
```

Comment:
It's good practice for non-trivial API methods (e.g. those which make HTTP requests, database queries, remote calls, ...) to take in a `context.Context` as first argument. This allows the client to cleanly control cancellations, timeouts, etc. I strongly recommend following this pattern in the SDK. For this particular call, take a look at [`http.NewRequestWithContext`](https://golang.org/pkg/net/http/#NewRequestWithContext).